ai_genomics/analysis/ai_genomics_experts/README.md

```diff
@@ -0,0 +1,32 @@
+# AI Genomics expert people and organisations
```

Comment:
actually - could be good to add the numbers of people/orgs per dataset in the readme?